dashboard/app/models/plc/task.rb

```diff
@@ -17,4 +19,16 @@ class Plc::Task < ActiveRecord::Base
   belongs_to :plc_learning_module, class_name: '::Plc::LearningModule'
   has_many :plc_task_assignments, class_name: '::Plc::EnrollmentTaskAssignment', foreign_key: 'plc_task_id', dependent: :destroy
   has_many :plc_evaluation_answers, class_name: '::Plc::EvaluationAnswer', foreign_key: 'plc_task_id', dependent: :destroy
+
+  include SerializedProperties
+  include StiFactory
+
+  # Can be overridden by subclasses, otherwise they will default to something like Learning Resource Task
+  def titleized_task_name
```

Comment:
oh I see this is relating to the name of the type of task, not the name of the task (`task.name`). In that case maybe call it something like `titleized_task_type` or `underscored_task_type` to avoid confusion with `task.name`?